doc/tutorial/climada_util_calibrate.ipynb

```diff
@@ -2512,6 +2485,65 @@
     "output_eval.plot_impf_set()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we show how the variability in parameter combinations with similar cost function values (as seen in the plot of the parameter space) translate to varying impact functions. In addition, the hazard value distribution is shown. Together this provides an intuitive overview regarding the robustness of the optimization, given the chosen cost function. It does NOT provide a view of the sampling uncertainty (as e.g. bootstrapping or cross-validation) NOR of the suitability of the cost function which is chosen by the user."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#select all hazard values at Exposure points\n",
+    "haz_vals = output_eval.input.hazard.intensity[:,output_eval.input.exposure.gdf.centr_TC]"
```

Comment:
If I understood correctly, calling the method `.plot_impf_variability` with `plot_haz=True` would do this step automatically, i.e., no need to provide the `haz_vals`. 

Do you perform this step explicitly here to show the functionality? Consider removing this step to illustrate the most "straight-forward" usages of `.plot_impf_variability`. i.e., do not encourage users to perform unnecessary steps.

Reply:
The haz_vals agrument was added for faster repeated plotting. However, with the newest version this is no longer required as now the default is fast enough. Thus, 'haz_vals' will be removed